server/service/src/sync/sync_status/status.rs

```diff
@@ -80,23 +96,23 @@ pub fn get_latest_sync_status(
         prepare_initial_done_datetime,
         push_start_datetime,
         push_done_datetime,
-        push_progress_start,
+        push_progress_total,
         push_progress_done,
         pull_central_start_datetime,
         pull_central_done_datetime,
-        pull_central_progress_start,
+        pull_central_progress_total,
         pull_central_progress_done,
         pull_remote_start_datetime,
         pull_remote_done_datetime,
-        pull_remote_progress_start,
+        pull_remote_progress_total,
         pull_remote_progress_done,
         integration_start_datetime,
         integration_done_datetime,
         error_message,
-    } = SyncLogRowRepository::new(&connection).load_latest_sync_log()?;
+    } = SyncLogRowRepository::new(&ctx.connection).load_latest_sync_log()?;
 
     let result = FullSyncStatus {
-        is_syncing: done_datetime.is_some() || error_message.is_some(),
+        is_syncing: done_datetime.is_none() && error_message.is_none(),
```

Comment:
:nit I'm getting confused between the usage of `done` as a counter of completed items and `done_datetime` which is, I think, completed datetime. the word `done` is being a bit flexible here

Reply:
How about ?

in database:
start_datetime -> started_datetime
done_datetime -> finished_datetime 

in graphql: 
total_progress -> total
done_progress -> done (in graphql can comment in schema, which will be `done`)

Reply:
291d72aa0f3f96df67c9a944ad6b066c42ffe148